homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -96,6 +103,14 @@
     }
 )
 
+SCHEMA_DUMP_HAP_CONFIG = vol.Schema(
+    {
+        vol.Required(ATTR_CONFIG_OUTPUT_PATH, default=""): cv.string,
```

Comment:
I think we should make this optional without default. Use dict.get to return `None` as default in the service handler.

Reply:
Done